otwcode/otwarchive
# AO3-4709 Comment freezing fixes

## Issue

https://otwarchive.atlassian.net/browse/AO3-4709

## Purpose

* Stops checking comments for spam whenever they are saved. This will allow spam comments to be frozen/unfrozen and prevent errors when you freeze/unfreeze the same comments repeatedly for testing purposes. It will also prevent comments you manually marked as spam from getting reverted by the spam checker.
* Removes the `cursor: pointer` CSS style that was being applied to all manner of things. Now we defer to whatever cursor the browser chooses for specific elements. (This will stop the pointer from showing up on "Frozen" and other things with the `.current` style, incorrectly making it look clickable. It also means some clickable things like buttons will now have the default arrow cursor, because that's what browsers do for those.)
* Update the redirect when freezing and unfreezing comments. Simply redirecting back would result in the comment section collapsing if you'd loaded comments with AJAX. So now we redirect to all comments, expanded, like when you mark something as ham/spam. AO3-6036 is the existing issue for it not redirecting you to the proper page.


## Testing Instructions

* Mark a comment as spam and make sure it doesn't get unmarked as spam when you freeze or unfreeze it.
* Make sure you don't get an error when repeatedly freezing and unfreezing the same comment.
* Make sure the "Frozen" indicator doesn't have the same cursor you see when hovering over a link.
* Make sure you get redirected to the comment's ultimate parent with comments shown when freezing/unfreezing.
